### ISSUE TITLE
[MC Version]:  Can't craft vanilla MC furnace (Incopat with Sawmill mod)

### ISSUE THREAD

**Issue by GamerTable** [DragonsPlusMinecraft/NethersDelightRefabricated#9]
### Description

Furnace has no recipe, can't craft it and is blank if you check with a mod like REI or JEI

### Steps to reproduce

1. Open game
2. Place cobblestone on crafting table...

### Mod list

Farmer's Delight Refabricated 1.20.1-2.1.1
Nether's Delight Refabricated 1.20.1-4.1.1
Fabric API 0.92.2+1.20.1
Moonlight 1.20-2.11.30
Sawmill 1.20-1.4.1

### Logs

https://mclo.gs/8xLktSM

### Forge

- [X] I am using the Forge version of Nether's Delight

### Minimal instance

- [X] I have tested this on a minimal instance

### Performance and shader mods

- [X] I have tested this without performance or shader mods

**Comment by arctlw** [comment]
Yup, there's some code somewhere that removes the recipe using blackstone but it seems to have removed all types of stone.

**Comment by shinycolororange** [comment]
yep same here took me some time to find the problem

**Comment by tiigerbones** [comment]
its not even a code, its inside the recipe folder, ray why the hell does ur mod remove vanilla furnace recipe brother

**Comment by DarkoPendragon** [comment]
> its not even a code, its inside the recipe folder, ray why the hell does ur mod remove vanilla furnace recipe brother

I'm necro posting a bit here, but its KubeJS that's interrupting the furnace.json file as a new recipe and overriding the actual recipe. The Nethers Delight mod by itself won't do this. A mod that uses KubeJS will cause this to happen.
